gluonnlp/data/transforms.py

```diff
@@ -326,3 +327,149 @@ def __call__(self, sample, return_str=False):
             List of words or detokenized text
         """
         return self._detokenizer.detokenize(sample, return_str=return_str)
+
+
+class JiebaTokenizer(object):
+    r"""Apply the jieba Tokenizer.
+
+    Users of this class are required to `install jieba <https://github.com/fxsjy/jieba>`_
+
+    Parameters
+    ----------
+    lang : str
+        The language to tokenize. Default is "zh", i.e, Chinese.
+
+    Examples
+    --------
+    >>> tokenizer = JiebaTokenizer()
+    >>> tokenizer(u"我来到北京清华大学")
+    ['我',
+     '来到',
+     '北京',
+     '清华大学']
+    >>> tokenizer(u"小明硕士毕业于中国科学院计算所，后在日本京都大学深造")
+    ['小明',
+     '硕士',
+     '毕业',
+     '于',
+     '中国科学院',
+     '计算所',
+     '，',
+     '后',
+     '在',
+     '日本京都大学',
+     '深造']
+
+    """
+    def __init__(self):
+        try:
+            import jieba
```

Comment:
this should be added to our extra dependencies https://github.com/dmlc/gluon-nlp/blob/master/setup.py#L55-L60